content/news/XXX-XX-XXX

```diff
@@ -0,0 +1,41 @@
+title: Mixxx 2.4.1 released
```

Comment:
Oh I see my comment was misleading, sorry. 
Please rename the file to `XXXX-XX-XX-mixxx-2-4-1-released.md`